main/test/ca/uwaterloo/flix/language/TestFlixErrors.scala

```diff
@@ -26,7 +26,7 @@ class TestFlixErrors extends FunSuite with TestUtils {
   def expectRuntimeError(v: Validation[CompilationResult, CompilationMessage], name: String): Unit = {
     expectSuccess(v)
     v match {
-      case Validation.Success(t) => t.getMain match {
+      case Validation.Success(t, _) => t.getMain match {
```

Comment:
Should this be changed to `Validation.Success(t, e) if e.isEmpty => ...`? and then handle non-empty errors like failures?

Reply:
Yes, this is the main concern. How we can be sure to do this right everywhere.